app/authentication/token.py

```diff
@@ -93,19 +88,34 @@ async def is_app_admin(token: str, app: str, error_str: str) -> bool:
         return True
 
 
-async def get_user(token: str = Depends(oauth2_scheme)) -> Tuple[str, str]:
-    """Calls GFW API to authorize user.
-
-    This functions check is user of any level is associated with the GFW
-    app and returns the user ID
-    """
+async def get_user(token: str = Depends(oauth2_scheme)) -> User:
+    """Get the details for authenticated user."""
 
     response: Response = await who_am_i(token)
 
-    if response.status_code == 401 or not (
-        "gfw" in response.json()["extraUserData"]["apps"]
-    ):
+    if response.status_code == 401:
         logger.info("Unauthorized user")
         raise HTTPException(status_code=401, detail="Unauthorized")
     else:
-        return response.json()["id"], response.json()["role"]
+        return User(**response.json())
```

Comment:
Note that the response from ResourceWatch has extra fields other than those in User, such as `_id`, `updatedAt`, and `provider`.  Will those extra keyword parameters to User work with the constructor of User, which is a pydantic model?  Just mentioning, in case that could be an issue.  But I see it seemed to be already working with the Signup class in the signup(() function in rw_api.py.

Reply:
I'm pretty sure it just ignore those extra parameters, and that's how it was already working, but good to note